zeek-config.h.in

```diff
@@ -248,19 +248,10 @@
 /* String with extension of dynamic libraries (e.g., ".so") */
 #define DYNAMIC_PLUGIN_SUFFIX "@CMAKE_SHARED_MODULE_SUFFIX@"
 
-/* Set if we're building outside of the main Zeek source code tree. */
-#ifndef ZEEK_PLUGIN_INTERNAL_BUILD
-#define ZEEK_PLUGIN_INTERNAL_BUILD true
```

Comment:
I was forth and back to say we should keep this, but I found that we removed the define `ZEEK_PLUGIN_INTERNAL_BUILD=false` for external plugins during the re-work already: https://github.com/zeek/cmake/commit/a9146d635357ef61cbf0e637df6d678d28e4a702#diff-7d1e0599dcdb8f6a71c933643b3d90adb768c608f62dc87067d847dd9b703036L242

So this is probably behaving wrongly with the current state for external plugins anyhow (?) and if the fix is to removes the define, it's a :+1: from me.